commands.js

```diff
@@ -64,6 +64,28 @@ class Commands {
     selfDestructMessage(() => message.reply('added time to user.'))
   }
 
+  async bitrate (message) {
+    let args = message.split(' ').splice(1).filter(str => str !== '')
+    let channel = message.member.voiceChannel
+    if (channel == null) {
+      throw new Error(`<@${message.author.id}>! You aren't in a voice channel`)
+    }
+    if (args.length === 0) {
+      selfDestructMessage(() => message.reply(`<#${channel.id}> bitrate: ${channel.bitrate}kbps`))
+    } else {
+      let newrate = parseInt(args[0])
+      if (isNaN(args[0]) || newrate < 8 || newrate > 384) {
```

Comment:
nit: check newrate instead of `args[0]` for NaN here.

Reply:
im checking isNaN cuz parseInt parses things like '1282aenrb34h' to 1282
its like /([0-9]+).*/